src/cli.ts

```diff
@@ -73,31 +346,14 @@ const app = command({
       process.exit(1);
     }
 
-    console.log(`Started migration.`);
-    const migGen = migrateGoogleDirGen({
-      googleDir,
-      outputDir,
-      errorDir,
-      warnLog: console.error,
-      exiftool: new ExifTool({ taskTimeoutMillis: timeout }),
-      endExifTool: true,
-    });
-
-    const counts = { err: 0, suc: 0 };
-    for await (const result of migGen) {
-      if (result instanceof Error) {
-        console.error(`Error: ${result}`);
-        counts.err++;
-        continue;
-      }
-
-      counts.suc++;
-    }
-
-    console.log(`Done! Processed ${counts.suc + counts.err} files.`);
-    console.log(`Files migrated: ${counts.suc}`);
-    console.log(`Files failed: ${counts.err}`);
+    const exifTool = new ExifTool({ taskTimeoutMillis: timeout });
+    await runBasicMigration(googleDir, outputDir, errorDir, exifTool);
   },
 });
 
+const app = subcommands({
+  name: 'google-photos-migrate',
+  cmds: { fullMigrate, folderMigrate, rewriteAllTags },
```

Comment:
No mention of this `rewriteAllTags` subcommand anywhere.  Add it to the README?

Reply:
This is currently only for testing purposes. The user doesn't need access to it.